strings.IndexByte not yet implemented

When trying to call `strings.Contains()` I get the following message:

```shell
/tmp/tinygo124212217/main.o: In function `strings.Index':
../../../../../../../gos/go1.11/src/strings/strings.go:995: undefined reference to `strings.IndexByte'
```

Does this mean that the `strings.IndexByte()` function is not yet implemented in TinyGo? If so, then where exactly should it be added?

--- a/src/runtime/string.go
+++ b/src/runtime/string.go
@@ -166,3 +166,14 @@ func decodeUTF8(s string, index uintptr) (rune, uintptr) {
 		return 0xfffd, 1
 	}
 }
+
+// indexByte returns the index of the first instance of c in s, or -1 if c is not present in s.
+//go:linkname indexByte strings.IndexByte
+func indexByte(s string, c byte) int {
+	for i := 0; i < len(s); i++ {
+		if s[i] == c {
+			return i
+		}
+	}
+	return -1
+}

